packages/core/bootstrap/src/lib/cache-warmer/util.ts

```diff
@@ -12,3 +15,106 @@ export function getSubscriptionKey(
     conf.hashOpts,
   )
 }
+
+export function splitIntoBatches(requestData: SubscriptionData): {
+  [path: string]: string[]
```

Comment:
Could clean up this file a lot and make it easier to grok by making a couple types that are re-used.

Reply:
so just create a type for `{ [path: string]: string[] }`?

Reply:
There are a lot of types in there that seem to get re-used a lot.

Seems like smallest pieces is something like:

`type BatchPaths = string[]`

which gets used in 
```
interface BatchRequestChunk {
[path: string]:  BatchPaths[]
}
```

which gets used in 

```
interface BatchesByPath {
[path: string]:  BatchRequestChunk[]
}
```